bowtie/report/public/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Bowtie –</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"
```

Comment:
Since there is now a bundling process for UI, maybe *bootstrap* could be put as a dependency in `package.json` file?

Reply:
I have used cdn in order to keep the application lightweight. (Also it might increase the loading speed because of caching).

Reply:
I mean, either way you still gonna need to download the same amount of bytes. I can see few advantages of putting it in package.json:
- you have all your dependencies in one place - it's easier to manage and e.g. dependabot could see this dependencies
- not relying on 3rd party sites - we don't need to worry this URL stopping to work (or changing) spontaneously. Each UI build/bundle would be independent and it's behaviour should be reproducible
- bundle produced by UI could also be cached as a whole or if it's necessary it can be split to chunks with different cache mechanics for each chunk.